packages/core/src/storybook/StorybookProvider.tsx

```diff
@@ -35,5 +35,8 @@ export const StorybookProvider: React.FunctionComponent<{
     return null
   }
 
+  // default to a non-existent user at the beginning of each story
+  mockCurrentUser(null)
```

Comment:
I usually like to be explicit for things like this.

```suggestion
  window.mockCurrentUser(null)
```

Reply:
Just so that people reading the code know that this is a global.

Reply:
Hmm, problem. Putting `window.` or `global.` in front of that function throws an error at build time:

```
@redwoodjs/core: src/storybook/StorybookProvider.tsx(39,10): error TS2339: Property 'mockCurrentUser' does not exist on type 'Window & typeof globalThis'.
```

Reply:
I'm assuming you're offline for the day and we wanna get 0.22 out, so I'll keep this as just `mockCurrentUser(null)` for now and we can revisit!